onedal/spmd/basic_statistics/incremental_basic_statistics.py

```diff
@@ -0,0 +1,78 @@
+# ==============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from daal4py.sklearn._utils import get_dtype
+
+from ..._device_offload import support_usm_ndarray
+from ...basic_statistics import (
+    IncrementalBasicStatistics as IncrementalBasicStatistics_nonSPMD,
```

Comment:
```suggestion
    IncrementalBasicStatistics as IncBS_Standard,
```

Just as a note, this is setting a precedent with the naming convention for incremental SPMD inheritance and class definition (similar to how ```_Batch``` has been previously used, but is not correct in this circumstance). Are we sure defining something due to the absence of an attribute is a good thing? Wouldn't something like ```_Standard``` or ```_Orig``` be better?  I know this is a ridiculous semantic thing, but we are going to see this now in 4+ spots through the codebase. @ethanglaser what are your thoughts?  If we shorten it, it may be simpler to read since we can reduce multi-lining.